x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/multivalue/MvMinTests.java

```diff
@@ -38,7 +38,7 @@ public static Iterable<Object[]> parameters() {
         longs(cases, "mv_min", "MvMin", (size, values) -> equalTo(values.min().getAsLong()));
         unsignedLongs(cases, "mv_min", "MvMin", (size, values) -> equalTo(values.reduce(BigInteger::min).get()));
         dateTimes(cases, "mv_min", "MvMin", (size, values) -> equalTo(values.min().getAsLong()));
-        return parameterSuppliersFromTypedDataWithDefaultChecks(false, cases);
+        return parameterSuppliersFromTypedDataWithDefaultChecks(false, cases, (v, p) -> "representableNonSpatial");
```

Comment:
I think this message is wrong (?), shouldn't it be human-readable?

Reply:
It is wrong, but let's grab it in a follow up, I think.

Reply:
yes, it should.  But again, this is just putting the string into the test, not changing the string that the function currently sends on type error.  I do think it's worth spending some time to review our type errors and update them, but that's not the goal of this PR.